libs/perun/services/src/lib/notificator.service.ts

```diff
@@ -10,8 +10,8 @@ export class NotificatorService {
   @Output() addNotification: EventEmitter<NotificationData> = new EventEmitter<NotificationData>();
   defaultAction: string;
   defaultRpcMessage: string;
-  defaultErrorDelayMs = 5_000;
-  defaultSuccessDelayMs = 3_000;
+  defaultErrorDelayMs = 7_000;
+  defaultSuccessDelayMs = 7_000;
```

Comment:
After testing it we could put it even higher, users seem to struggle with it, so 2sec increase might not be sufficient.
```suggestion
  defaultErrorDelayMs = 10_000;
  defaultSuccessDelayMs = 10_000;
```